### PR TITLE
Add Travis YML

### PR DESCRIPTION
**Pull request by FokkeZB**
Once this is merged Travis should be enabled for this repo. It will ensure the library compiles.

### CODE REVIEW

**Comment by FokkeZB** [comment]
@johanstokking kun je deze mergen?

File: .travis.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+language: c
+sudo: false
+before_install:
+  - source <(curl -SLs https://raw.githubusercontent.com/adafruit/travis-ci-arduino/master/install.sh)
+script:
+  - build_main_platforms
+  - build_platform trinket
```
Comment (johanstokking) [inline .travis.yaml]: Add newline, also instruct your editor to add that automatically always

**Comment by FokkeZB** [comment]
@htdvisser can you enable?

**Comment by htdvisser** [comment]
https://travis-ci.org/TheThingsNetwork/arduino-device-lib

**Pull request merged**